rocrand/PKGBUILD

```diff
@@ -2,28 +2,29 @@
 pkgname=rocrand
 pkgver=3.3.0
 pkgrel=1
-pkgdesc="RAND library for HIP programming language"
+pkgdesc='Pseudo-random and quasi-random number generator on ROCm'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/rocRAND"
 license=('MIT')
-makedepends=("hcc>=$pkgver" 'cmake')
-source=("https://github.com/ROCmSoftwarePlatform/rocRAND/archive/rocm-$pkgver.tar.gz")
+depends=('hip-hcc')
+makedepends=('hcc' 'cmake' 'git')
+source=("$url/archive/rocm-$pkgver.tar.gz")
```

Comment:
Can we have ${pkgname}-${pkgver} to the source? See: https://wiki.archlinux.org/index.php/PKGBUILD#source

Reply:
Good point. I've changed it.